code/controllers/subsystem/processing/quirks.dm

```diff
@@ -30,6 +30,7 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 		list("Quadruple Amputee", "Paraplegic"),
 		list("Quadruple Amputee", "Frail"),
 		list("Social Anxiety", "Mute"),
+		list("Mute", "Soft-Spoken"),
```

Comment:
Bad aim and big hands should probably be mutually exclusive

Reply:
yes